jenkins/run

```diff
@@ -54,12 +46,12 @@ fi
 podman run \
     --pull=always \
     --userns=keep-id \
-    --volume $(pwd):/src/pbench:z \
-    --volume ${HOME_VOLUME} \
+    --volume $(pwd):${HOME_DIR}/pbench:z \
+    --volume ${HOME_DIR} \
     ${GIT_BASE_VOLUME} \
-    -w /src/pbench \
-    --env-host \
-    --env HOME=${IHOME} \
+    -w ${HOME_DIR}/pbench \
+    --env HOME=${HOME_DIR} \
+    --env USER=${USER_NAME} \
```

Comment:
Are you sure that we want to override the value of `USER`?  I think this will mean that the `uid` (e.g., as returned by `id -nu`) will disagree with the value in `USER`, which seems like it will cause confusion.  Does this accomplish anything other than to match `USER` to the last directory in `HOME` (which might be nice but hardly seems important...)?

Reply:
`USER` is no longer passed in.

Reply:
What does line 54 do?  It looks to me like it defines the `USER` environment variable inside the container.

Reply:
See future commits.

Reply:
So this commit is landed by GitHub across multiple commits.  So I am not entirely sure what version this comment is made against.

Considering the code compared against `main`, by dropping `--env-host`, which leaks all kinds of stuff into the container environment, we need a `USER` for certain programs and sub-systems to work.